avm/ptn/app-service-lza/hosting-environment/modules/front-door/front-door.module.bicep

```diff
@@ -0,0 +1,295 @@
+@description('Required. Name of the AFD profile.')
+param afdName string
+
+@description('Name of the endpoint under the profile which is unique globally.')
+param endpointName string
+
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+@description('AFD Endpoint State')
+param endpointEnabled string = 'Enabled'
+
+@description('Optional. Endpoint tags.')
+param tags object = {}
+
+@allowed([
+  'Standard_AzureFrontDoor'
+  'Premium_AzureFrontDoor'
+])
+@description('Required. The pricing tier (defines a CDN provider, feature list and rate) of the CDN profile.')
+param skuName string
+
+// @description('Custom Domain List')
+// param customDomains array
+
+@description('The name of the Origin Group')
+param originGroupName string
+
+@description('Origin List')
+param origins array
+
+@allowed([
+  'Block'
+  'Log'
+  'Redirect'
+])
+param wafRuleSetAction string = 'Block'
+
+@description('optional, default value Enabled. ')
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+param wafPolicyState string = 'Enabled'
+
+@description('optional, default value Prevention. ')
+@allowed([
+  'Detection'
+  'Prevention'
+])
+param wafPolicyMode string = 'Prevention'
+
+@description('Required. The resource ID of the Log Analytics workspace to send logs to.')
+@secure()
+param logAnalyticsWorkspaceId string
```

Comment:
Just a quick call out that I noticed that you use both `logAnalyticsWorkspaceId` and `logAnalyticsWId`. I'd recommend to stick with one - and the former :wink: